frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -53,14 +53,19 @@
 .truncate {
     text-overflow: ellipsis;
     overflow: hidden;
+    padding-right: 50px;
+    position: relative;
     white-space: nowrap;
 }
 
 .metadata button {
     background: none;
     border: none;
     color: #7BC876;
-    float: right;
+    font-style: italic;
     font-weight: 300;
     padding: 0;
+    position: absolute;
+    top: 3px;
```

Comment:
I think this should be 2px. It would be even more noticeable if the `font-style` would `normal`.